src/NUnitFramework/tests/Throws.cs

```diff
@@ -0,0 +1,136 @@
+// ****************************************************************
+// Copyright 2009, Charlie Poole
+// This is free software licensed under the NUnit license. You may
+// obtain a copy of the license at http://nunit.org
+// ****************************************************************
+
+using System;
+using System.Collections;
+using NUnit.Framework.Constraints;
+
+namespace NUnit.Framework
+{
+    /// <summary>
+    /// Helper class with properties and methods that supply
+    /// constraints that operate on exceptions.
+    /// </summary>
+    public class Throws
```

Comment:
Why isn't this class `static`?

Is this being added in this PR because of the compatibility recommendation?

Reply:
Should be! 

Yes! I'm happy to say that only a very few of the recommendations are currently unavailable in 2.6.5. The messages indicate which ones.